CONTRIBUTING.md

```diff
@@ -1,4 +1,7 @@
-# To Fork or Not To Fork?
+# Contributor Contributing Notes

```

Comment:
This is definitely in the area of personal preference, but I'd like to suggest making the header "Contributing Guidelines". That's usually what's used to refer to the contents of this document, including in various GitHub features.